src/errormap.jl

```diff
@@ -1,6 +1,6 @@
-	# Error maps (maps err to (NAME, Description))
+        # Error maps (maps err to (NAME, Description))
 
-const error_codes = {
+const error_codes = Dict (
```

Comment:
So is the idea that you use the most current syntax and Compat takes care of the rest, magically in the background?